cupy/linalg/eigenvalue.py

```diff
@@ -121,7 +121,10 @@ def eigh(a, UPLO='L'):
 
     .. seealso:: :func:`numpy.linalg.eigh`
     """
-    return _syevd(a, UPLO, True)
+    if a.ndim == 3:
+        return cupy.cusolver._syevj_batched(a, UPLO, True)
```

Comment:
There is an issue with removing the `cupy.` prefix here since `_syevj_batched` is implemented in `cupy.cusolver` and the `cusolver` reference in this file is to `cupy.cuda.cusolver`. Do you have a suggestion for resolving this issue?

Reply:
Ah, okay. We may use an alias to import, but I think your original lines are enough.

Reply:
Okay, I committed this suggestion before realizing the issue. I will push a fix shortly.